ServiceStation/ConsoleApp/Menu.cs

```diff
@@ -0,0 +1,392 @@
+﻿using BusinessLayer.Models.DTO;
+using BusinessLayer.Services.ServiceStationService;
+using BusinessLayer.ServiceStationService;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ConsoleApp
+{
+    public class Menu
+    {
+        private readonly CarService car;
+
+        private readonly CarTechnicalConditionService carCondition;
+
+        private readonly OwnerService owner;
+
+        private readonly InspectorService inspector;
+
+        public Menu(CarService car, CarTechnicalConditionService carCondition,
+                                OwnerService owner, InspectorService inspector)
+        {
+            this.car = car;
+            this.carCondition = carCondition;
+            this.owner = owner;
+            this.inspector = inspector;
+        }
+
+        public async Task Start()

```

Comment:
Split menu by entities.
it's very hard to review.

Reply:
Hm.
You not understand me)
You should split one menu class into few menus classe for existing entities.
Each menu class will be responsible only for the one entity.
For example, StationMenu class will provide menu for stations and all operations will be in it.

Reply:
Fixed